libs/host/Configuration/Options.cs

```diff
@@ -664,8 +664,8 @@ private IAuthenticationSettings GetAuthenticationSettings(ILogger logger = null)
                     var aadAuthSettings = new AadAuthenticationSettings(AuthorizedAadApplicationIds?.Split(','), AadAudiences?.Split(','), AadIssuers?.Split(','), IssuerSigningTokenProvider.Create(AadAuthority, logger), AadValidateUsername.GetValueOrDefault());
                     return new AclAuthenticationAadSettings(AclFile, Password, aadAuthSettings);
                 default:
-                    logger?.LogError("Unsupported authentication mode: {mode}", AuthenticationMode);
-                    throw new Exception($"Authentication mode {AuthenticationMode} is not supported.");
+                    logger?.LogWarning("Defaulting to NoAuth if authenticationSettingsOverride is not provided.");
```

Comment:
why is this needed? you can send NoAuth and provide override.

Reply:
Resolved, Thanks